docker-style/action.yml

```diff
@@ -0,0 +1,157 @@
+name: >
+  Lint Dockerfile
+
+description: >
+  Evaluate the quality of your project Dockerfile(s) by using `hadolint
+  <https://github.com/hadolint/hadolint/>`_. This action can be used to
+  lint Dockerfile(s) from multiple directories, see input "directory"
+  description. The action uses `hadolint-gh-action
+  <https://github.com/jbergstroem/hadolint-gh-action>`_ behind the scenes.
+  If you want to evaluate multiple Dockerfiles contained in various
+  directories of the provided directory, use the recursive option.
+  When linting a Dockerfile dedicated to Windows, one should use hadolint
+  shell pragma to avoid false positives from ShellCheck, see `hadolint shell pragma
+  <https://github.com/hadolint/hadolint/pull/708>`_.
+
+  .. note::
+
+      This action emphasizes the fact of having Dockerfile(s) contained inside the
+      'docker' directory in the root of the project.
+
+  .. warning::
+
+      This action only looks for docker files named Dockerfile.
+      A docker file like Dockerfile.linux will not be linted.
+
+inputs:
+
+  # Optional inputs
```

Comment:
Should we make these required inputs?

Reply:
I'm not opposed to it. However, I liked the fact that we pointed to the docker directory by default. It reinforces the idea that this is 'the default place'.

Reply:
Sounds good.